blocks/footer/footer.js

```diff
@@ -1,487 +1,15 @@
-
-    const popular = {
-      title: popularTitle.textContent,
-      items: toItems(popularItems),
-    };
-
-    const aboutTitle = doc.querySelector('div > p:nth-of-type(3)');
-    const aboutItems = doc.querySelectorAll('div > p:nth-of-type(3) + ul a');
-
-    const about = {
-      title: aboutTitle.textContent,
-      items: toItems(aboutItems),
-    };
-
-    const socialTitle = doc.querySelector('div > p:nth-of-type(4)');
-    const socialItems = doc.querySelectorAll('div > p:nth-of-type(4) + ul a');
-
-    const social = {
-      title: socialTitle.textContent,
-      items: toItems(socialItems),
-    };
-
-    const zeissTitle = doc.querySelector('div > p:nth-of-type(5)');
-    const zeissItems = [
-      doc.querySelectorAll('div > p:nth-of-type(5) + ul a'),
-      doc.querySelectorAll('div > p:nth-of-type(5) + ul + ul a'),
-      doc.querySelectorAll('div > p:nth-of-type(5) + ul + ul + ul a'),
-    ];
-
-    const zeiss = {
-      title: zeissTitle.textContent,
-    };
-    const zeissLocation = path2label(currentPaths[1]);
-    zeissItems.forEach((elements, index) => {
-      zeiss[`items${index}`] = toItems(elements);
-
-      if (zeiss[`items${index}`].some((item) => item.label === zeissLocation)) {
-        zeiss.selected = zeissLocation;
-      }
-    });
-
-    const languageTitle = doc.querySelector('div > p:nth-of-type(6)');
-    const languageGlobalTitle = doc.querySelector('div > p:nth-of-type(7)');
-    const languageEN = doc.querySelector('div > p:nth-of-type(7) + ul a');
-    const languageEuropaTitle = doc.querySelector('div > p:nth-of-type(8)');
-    const languageDE = doc.querySelector('div > p:nth-of-type(8) + ul a');
-
-    const isDE = currentPaths[0] === 'de';
-
-    const language = {
-      title: languageTitle.textContent,
-      selected: isDE ? languageDE.textContent : languageEN.textContent,
-      global: {
-        title: languageGlobalTitle.textContent,
-        // TODO currently hardcoded in the content
-        en: {
-          label: languageEN.textContent,
-          href: languageEN.href,
-        },
-      },
-      europa: {
-        title: languageEuropaTitle.textContent,
-        // TODO currently hardcoded in the content
-        de: {
-          label: languageDE.textContent,
-          href: languageDE.href,
-        },
-      },
-      items: toItems(socialItems),
-    };
-
-    const legalTitle = doc.querySelector('div > p:nth-of-type(9)');
-    const legalItems = doc.querySelectorAll('div > p:nth-of-type(9) + ul a');
-
-    const legal = {
-      title: legalTitle.textContent,
-      items: toItems(legalItems),
-    };
-
-    block.innerHTML = template({
-      nav,
-      relatedSites,
-      popular,
-      about,
-      social,
-      zeiss,
-      language,
-      legal,
-    });
-
-    decorateIcons(block, true);
-
-    addFooterInteractions(block);
-  }
+  const footerUrl = 'https://www.zeiss.com/semiconductor-manufacturing-technology/home.html';
```

Comment:
@sdmcraft how's the langage detected ? 

We could either use the path root 

   ```
   /de/semiconductor-manufacturing... > de
   ```
or the FQDN root

   ```
   www.zeiss.de > de
   ```
If possible we'd use the former as it will behave properly in a local development environment.

Reply:
I had an alternative idea which should work for both situations if we add it on the serverside via metadata spreadsheet. [More details here.](https://cq-dev.slack.com/archives/C04RN3CQWBZ/p1680769192479689)

Reply:
Done, thank you